packages/lib/src/utils/Transactions.ts

```diff
@@ -109,61 +116,60 @@ async function _sendTransaction(contractFn, args = [], txParams = {}) {
+    return contract.new(...args, txParams);
   }
 
-  const data = contract.getData(args, txParams)
-  const gas = await estimateActualGas({ data, ...txParams })
-  return contract.new(...args, { gas, ...txParams })
+  const data: string = contract.getData(args, txParams);
+  const gas: number = await estimateActualGas({ data, ...txParams });
+  return contract.new(...args, { gas, ...txParams });
 }
 
-export async function estimateGas(txParams, retries = RETRY_COUNT) {
+export async function estimateGas(txParams: any, retries: number = RETRY_COUNT): Promise<any> {
   // Retry if estimate fails. This could happen because we are depending
   // on a previous transaction being mined that still hasn't reach the node
   // we are working with, if the txs are routed to different nodes.
   // See https://github.com/zeppelinos/zos/issues/192 for more info.
   try {
     // Use json-rpc method estimateGas to retrieve estimated value
-    return await ZWeb3.estimateGas(txParams)
+    return await ZWeb3.estimateGas(txParams);
   } catch (error) {
-    if (retries <= 0) throw Error(error);
+    if (retries <= 0) { throw Error(error); }
     await sleep(RETRY_SLEEP_TIME);
     return await estimateGas(txParams, retries - 1);
   }
 }
 
-export async function estimateActualGasFnCall(contractFn, args, txParams, retries = RETRY_COUNT) {
+export async function estimateActualGasFnCall(contractFn: GenericFunction, args: any[], txParams: any, retries: number = RETRY_COUNT): Promise<any> {
   // Retry if estimate fails. This could happen because we are depending
   // on a previous transaction being mined that still hasn't reach the node
   // we are working with, if the txs are routed to different nodes.
   // See https://github.com/zeppelinos/zos/issues/192 for more info.
   try {
-    return await calculateActualGas(await contractFn.estimateGas(...args, txParams))
+    return await calculateActualGas(await contractFn.estimateGas(...args, txParams));
   } catch(error) {
-    if (retries <= 0) throw Error(error);
+    if (retries <= 0) { throw Error(error); }
     await sleep(RETRY_SLEEP_TIME);
     return estimateActualGasFnCall(contractFn, args, txParams, retries - 1);
   }
 }
 
-export async function estimateActualGas(txParams) {
-  const estimatedGas = await estimateGas(txParams)
-  return calculateActualGas(estimatedGas)
+export async function estimateActualGas(txParams: any): Promise<any> {
+  const estimatedGas = await estimateGas(txParams);
+  return calculateActualGas(estimatedGas);
 }
 
-
-async function getNodeVersion() {
+async function getNodeVersion(): Promise<string> {
   if (!state.nodeInfo) {
-    state.nodeInfo = await ZWeb3.getNode()
+    state.nodeInfo = await ZWeb3.getNode();
   }
   return state.nodeInfo;
 }
 
-async function getETHGasStationPrice() {
-  if (state.gasPrice) return state.gasPrice;
+async function getETHGasStationPrice(): Promise<any> | never {
+  if (state.gasPrice) { return state.gasPrice; }
```

Comment:
We should definitely review some tslint rules. Adding braces in a one-line statement is somewhat annoying.

Reply:
Agreed! Implemented:
`"curly": [true, "ignore-same-line"]`
"curly" also has the "as-needed" option, but I decided to not use it, because it would allow something like this, which I don't like:
```
if(blah)
  thenDoMeh();
```